src/Apps/Collect/Components/Collection/Header/DefaultHeader.tsx

```diff
@@ -0,0 +1,131 @@
+import { Box, color, Flex, Image } from "@artsy/palette"
+import { DefaultHeader_headerArtworks } from "__generated__/DefaultHeader_headerArtworks.graphql"
+import React, { FC } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import styled from "styled-components"
+import { useWindowSize } from "Utils/Hooks/useWindowSize"
+import { getViewportWidth } from "Utils/viewport"
+
+interface Props {
+  headerArtworks: DefaultHeader_headerArtworks
+  defaultHeaderImageHeight: number
+}
+
+const IMAGE_MARGIN_X = 10
+const LARGE_VIEWPORT_WIDTH = 880
+
+export const CollectionDefaultHeader: FC<Props> = ({
+  headerArtworks,
+  defaultHeaderImageHeight,
+}) => {
+  useWindowSize()
```

Comment:
I think you should be able to get the width from the return value here, and eliminate line 28 below (another call to `getViewportWidth`).